Leanplum/Leanplum.h

```diff
@@ -14,6 +14,33 @@ FOUNDATION_EXPORT double LeanplumVersionNumber;
 //! Project version string for Leanplum.
 FOUNDATION_EXPORT const unsigned char LeanplumVersionString[];
 
-// In this header, you should import all the public headers of your framework using statements like #import <Leanplum/PublicHeader.h>
 
+@interface Leanplum : NSObject
```

Comment:
If we make the Leanplum object have class functions, can we ensure that there is no state that we store in this class? If we do want to store state, we should do it in a different class and just call it from this one

Reply:
sure. We will keep that in mind. Once we have caching layer started.